src/Dockerfile

```diff
@@ -46,13 +46,14 @@ RUN cargo +1.49.0 install wasm-pack
 RUN cargo +1.49.0 install cargo-tree
```

Comment:
Why 1.49 here?

Reply:
Same here. I don't know why. The image is exactly as it was on docker-builder repo, but with another base image

Reply:
Yes, it is a good opportunity to review these things..

Reply:
Done!